.bazelrc

```diff
@@ -5,6 +5,18 @@ build --color=yes
 build --cxxopt=-std=c++14
 build --cxxopt=-g2
 
+build --javacopt="-source 8"
```

Comment:
I know this is a draft! But thought I would drop a question while Im' here.  Why the java stuff?

Reply:
Oh your PR description says why. Sad that it's required - how large an additional fetch is this?

Reply:
yes still trying to figure this out. But somehow the codecov doesn't work without java. It complains about not being able to find java :/ still trying to understand why though.

Reply:
good question.... can research and update the PR.